tests/test_user_common.sh

```diff
@@ -23,9 +23,7 @@ source ./tests/util/util_users.sh
 source ./tests/commands/list_buckets.sh
 
 test_admin_user() {
-  if [[ $# -ne 1 ]]; then
-    fail "test admin user command requires command type"
-  fi
+  assert [ $# -eq 1 ]
```

Comment:
Looks like you are removing the information associated with this failure?  Is that stored somewhere else for debugging?

Reply:
Looks like I initially set this up for different clients, but am not using them yet.  So I can just remove that check.